src/routes/(console)/organization-[organization]/billing/planSummary.svelte

```diff
@@ -1,180 +1,564 @@
 <script lang="ts">
     import { base } from '$app/paths';
-    import { CardGrid } from '$lib/components';
+    import { EstimatedCard } from '$lib/components';
     import { Button } from '$lib/elements/forms';
     import { toLocaleDate } from '$lib/helpers/date';
-    import { plansInfo, upgradeURL } from '$lib/stores/billing';
+    import { upgradeURL } from '$lib/stores/billing';
     import { organization } from '$lib/stores/organization';
-    import type { Aggregation, Invoice, Plan } from '$lib/sdk/billing';
-    import { abbreviateNumber, formatCurrency, formatNumberWithCommas } from '$lib/helpers/numbers';
+    import type { AggregationTeam, Invoice, Plan } from '$lib/sdk/billing';
+    import { formatCurrency } from '$lib/helpers/numbers';
     import { BillingPlan } from '$lib/constants';
     import { Click, trackEvent } from '$lib/actions/analytics';
-    import {
-        Accordion,
-        Card,
-        Divider,
-        Icon,
-        Layout,
-        Tooltip,
-        Typography
-    } from '@appwrite.io/pink-svelte';
-    import { IconInfo, IconTag } from '@appwrite.io/pink-icons-svelte';
+    import { Divider, Typography, Expandable as ExpandableTable } from '@appwrite.io/pink-svelte';
+    import { humanFileSize } from '$lib/helpers/sizeConvertion';
+    import { formatNum } from '$lib/helpers/string';
+    import { ProgressBar } from '$lib/components';
+    import { isSmallViewport, isTabletViewport } from '$lib/stores/viewport';
     import CancelDowngradeModel from './cancelDowngradeModal.svelte';
 
     export let currentPlan: Plan;
     export let currentInvoice: Invoice | undefined = undefined;
-    export let availableCredit: number | undefined = undefined;
-    export let currentAggregation: Aggregation | undefined = undefined;
+    export let currentAggregation: AggregationTeam | undefined = undefined;
+    export let organizationUsage: any = undefined;
+    export let usageProjects: Record<string, any> = {};
 
     let showCancel: boolean = false;
 
-    const today = new Date();
-    const isTrial =
-        new Date($organization?.billingStartDate).getTime() - today.getTime() > 0 &&
-        $plansInfo.get($organization.billingPlan)?.trialDays;
-    const extraUsage = currentInvoice ? currentInvoice.amount - currentPlan?.price : 0;
+    // define columns for the expandable table
+    const columns = [
+        { id: 'item', align: 'left' as const, width: '10fr' },
+        { id: 'usage', align: 'left' as const, width: '20fr' },
+        { id: 'price', align: 'right' as const, width: '0fr' }
```

Comment:
we don't use `fr` values for other table types, why here? if its like this on library, will need to change i think.